src/parse.rs

```diff
@@ -858,6 +816,8 @@ impl<'a> Bytes<'a> {
     }
 
     pub fn unsigned_integer<T: Num>(&mut self) -> Result<T> {
```

Comment:
Since `signed_integer` and `unsigned_integer` are still exposed, they should get extra tests with strong typing